.github/workflows/build-executor-win.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       #         Write-Host "Error: Directory not found: $vs2022Path"
       #     }
 
-      - name: Build
+      - name: Build & test
```

Comment:
Nit: Could you please update the name on of this workflow to `Build + Test Windows executor`?

I think we can leave the filename as is though